cmd/api-errors.go

```diff
@@ -1781,7 +1779,7 @@ func toAPIError(ctx context.Context, err error) APIError {
 			}
 		case crypto.Error:
 			apiErr = APIError{
-				Code:           "XKMSInternalError",
+				Code:           "XMinIOEncryptionError",
```

Comment:
Is Crypto instead of Encryption preferable in the name here? The error could be in decryption or something else too.

Reply:
Generally, Decryption is synonymous with Encryption meaning headers are same for decryption which is used for encryption @donatello